recipes/tornado/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version = "4.2.1" %}
+
+package:
+  name: tornado
+  version: {{ version }}
+
+source:
+  fn: tornado-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/source/t/tornado/tornado-{{ version }}.tar.gz
+  md5: d523204389cfb70121bb69709f551b20
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - ssl_match_hostname
+    - certifi
+  run:
+    - python
+    - ssl_match_hostname
+    - certifi
+
+test:
+  imports:
+    - tornado
+    - tornado.platform
+    - tornado.test
+
+about:
+  home: http://www.tornadoweb.org/
+  license: Apache Software License
+  summary: 'Tornado is a Python web framework and asynchronous networking library, originally developed at FriendFeed.'
+
+extra:
+  recipe-maintainers:
+    - jakirkham
```

Comment:
Honestly there is a bunch of Jupyter stuff (and related dependencies) that are crucial to what I do on a regular basis. I have no problem being added to those sorts of things. Though I certainly appreciate being asked, if it is notebook related, basically I'm on board with being a maintainer.